org.gnome.Radio.json

```diff
@@ -0,0 +1,105 @@
+{
+  "app-id" : "org.gnome.Radio",
+  "runtime" : "org.gnome.Platform",
+  "runtime-version" : "43",
+  "sdk" : "org.gnome.Sdk",
+  "command" : "gnome-radio",
+  "finish-args" : [
+     "--socket=x11",
+     "--share=ipc"
```

Comment:
it doesn't support wayland?

Reply:
I think it does, it builds against gtk4 which should support wayland as a backend, I think.

Reply:
The executable org.gnome.Radio supports gtk4 so it should support wayland as a backend.
The executable /usr/bin/gnome-radio (Champlain, Clutter and gtk3-based) works only with x11.

See https://gitlab.gnome.org/ole/gnome-radio/-/issues/2